### PR TITLE
Use standard file reading functions in input time series

### PR DESCRIPTION
**Pull request by r-ash**

### CODE REVIEW

File: R/input-time-series.R [hunk 1]
```diff
@@ -320,7 +320,7 @@ prepare_input_time_series_anc <- function(anc, shape) {
 
   ## Check if art is object or file path
   if(!inherits(anc, c("spec_tbl_df","tbl_df","tbl","data.frame" ))) {
-    anc <- readr::read_csv(anc, show_col_types = FALSE)
+    anc <- read_anc_testing(anc)
```
Comment (jeffeaton) [inline R/input-time-series.R:323]: Do you need the `area_name` from this one? (Does that get used in any of the time series code, etc.?)

**Review by rtesra — APPROVED** [review]

**Review by jeffeaton — APPROVED** [review]
Thanks, nice one.  I am approving this based on what is here looks good.

One thing to check: does the time series plots rely on the `area_name` at all?  
* Will not having the `area_name` field read by `read_anc_testing()` create any issue?
* Will having the `area_name` present in `read_art_number(..., all_columns = TRUE)` create any problem? (E.g. merging with another dataset that has `area_name`)

File: R/input-time-series.R (hunk 1 above)
Comment (rtesra) [reply R/input-time-series.R:323]: No `area_name` is pulled back in after merging on `area_id` with the boundaries file so not needed from the anc data:  https://github.com/mrc-ide/naomi/pull/283/files#diff-41ca6a0b90b4c5a022737ba75b3a161ccd69eaa2258feb8cb628ed2bd69733bdR30

**Comment by r-ash** [comment]
> Thanks, nice one. I am approving this based on what is here looks good.
> 
> One thing to check: does the time series plots rely on the `area_name` at all?
> 
>     * Will not having the `area_name` field read by `read_anc_testing()` create any issue?
> 
>     * Will having the `area_name` present in `read_art_number(..., all_columns = TRUE)` create any problem? (E.g. merging with another dataset that has `area_name`)

Thanks, have updated to remove `area_name` reading from `read_art_number`, `read_anc_testing` doesn't read it either.
Also noticed `aggregate_` functions which had a call to `read_csv` which I've removed in favour of the Naomi reading function too.

**Review by jeffeaton — APPROVED** [review]

File: R/input-time-series.R [hunk 2]
```diff
@@ -318,9 +318,10 @@ prepare_input_time_series_anc <- function(anc, shape) {
     areas <- shape %>% sf::st_drop_geometry()
   }
 
+  browser()
```
Comment (jeffeaton) [inline R/input-time-series.R:321]: Oops, unapproved!

**Review by jeffeaton — CHANGES_REQUESTED** [review]
Ack -- unapprove! There's a browser() call left in there.

File: R/input-time-series.R (hunk 2 above)
Comment (jeffeaton) [reply R/input-time-series.R]: I've just removed

**Review by jeffeaton — APPROVED** [review]

**Comment by jeffeaton** [comment]
I just added a NEWS entry for this version (should have picked that up on review too..oops, sorry).

File: R/input-time-series.R (hunk 2 above)
Comment (r-ash) [reply R/input-time-series.R]: Oops, good spot - thanks

**Pull request merged**